release notes/v0.5.0.md

```diff
@@ -0,0 +1,125 @@
+xk6-browser v0.5.0 is here! :tada:
+
+In this minor release we have attempted to make the application more stable by tackling the most problematic race conditions we have come across. The most notable changes are the following two **breaking changes**:
+
+- `click` and `waitForNavigation` are now asynchronous, although we do not support the `async` keyword yet. We plan to make more of our apis asynchronous in the next release.
+
+- We have a new `Chromium` exported `BrowserType` which is to be used to launch a new instance of the `chromium` browser, and `launch` is no longer supported.
+
+
+## Bugs fixed
+
+- `Page.goto()` race condition, which was mainly found to occur in CI, looks to have been fixed. ([#480](https://github.com/grafana/xk6-browser/pull/480))
+
+- Fix nil panic race in `FrameManager.waitForFrameNavigation`. ([#500](https://github.com/grafana/xk6-browser/pull/500), [#501](https://github.com/grafana/xk6-browser/pull/501), [#508](https://github.com/grafana/xk6-browser/pull/508))
+
+
+## Improvements
+
+- We've made more changes to out log messages to make them more concise and user friendlier. ([#438](https://github.com/grafana/xk6-browser/pull/438))
+
+- `Chromium` is now an importable type which is the defacto method of launching tests using `Chromium` (which is still the only supported browser type). This is a **breaking change** which supersedes the `launch` function. The same options are supported in `Chromium`'s `launch` method as in the previous `launch` function. ([#462](https://github.com/grafana/xk6-browser/pull/462), [#515](https://github.com/grafana/xk6-browser/pull/515))
```

Comment:
WDYT about moving this and the async API changes to a "Breaking changes" section?

They're not really improvements (well, async could be considered as such), but the biggest impact for users is the syntax change, so a separate section for that would make it more visible.

In which case, we can remove the summary from the introduction and just mention async APIs and breaking changes.

Reply:
Done in 74b363a. Let me know what you think.